.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea/
+.terraform.lock.hcl
+.terraform/
+backend_storage.tf
+main.py
```

Comment:
I'd remove main.py it should not be in the repository anyways

Reply:
done